shared/js/web_manifest_helper.js

```diff
@@ -160,7 +160,7 @@
           return true;
         }
         size = size.toLowerCase();
-        if (!size.contains('x') || size.indexOf('x') !==
+        if (!size.indexOf('x') || size.indexOf('x') !==
```

Comment:
This condition isn't correct anymore. If it was a clever attempt at making the size 'x123' invalid, that's neat, but this also makes a size with no 'x' valid, which I'm guessing isn't valid, and certainly isn't expected by the code that follows.

Really, this should probably just use a regex match, like `if (!size.match(/^\d+x\d+$/))`